image_op/norm_by_disk_filter.m

```diff
@@ -0,0 +1,24 @@
+function M_norm = norm_by_disk_filter(movie)
+%Normalize every frame in the movie by a disk filtered version of itself
+%
+%   movie: movie matrix , [h x w x num_frames]
+%
+% 2015 01 31 Tony Hyun Kim (Revised: Hakan Inan, 15-Jan-4)
+%
+
+M_norm = zeros(size(movie), 'single');
+num_frames = size(movie,3);
+% Apply spatial normalization
+hDisk = fspecial('disk', 15);
```

Comment:
The radius of the disk should be exposed as a function parameter (e.g. `disk_radius`).